.pre-commit-config.yaml

```
@@ -9,11 +9,19 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
```

Let's add a comment to this line as well

```suggestion
    rev: 4.0.1  # must match requirements-tests.txt
```